felix_requirements.txt

```diff
@@ -1,4 +1,4 @@
 gevent
 greenlet
 netaddr
-python-etcd>=0.3.3-calico-1
+python-etcd==0.3.3-calico-2
```

Comment:
Please skip the hard pin, it just causes us problems if we have to version rev again.